projects/sft-protocol/index.js

```diff
@@ -40,7 +44,11 @@ module.exports = {
     filecoin: {
         tvl: async (_, _1, _2, { api }) => {
 
-            let balances = {};
+            const totalPooledFILAmount = await api.call({ target: LENDING_POOL_ADDRESS, abi: GET_TOTAL_POOLED_FIL_ABI, params: [] });
+            const totalLendingFILAmount = await api.call({ target: LENDING_POOL_ADDRESS, abi: GET_TOTAL_LENDING_FIL_ABI, params: [] });
+            let lendingPoolTvl = totalPooledFILAmount - totalLendingFILAmount;
+            api.add(ADDRESSES.null, lendingPoolTvl);
+            let balances = await sumTokens2({ owner: LENDING_POOL_ADDRESS, tokens, api });
```

Comment:
since we are fetching fil in this line and adding it to balances, line 50 and above is redundant?

Reply:
my fault, the redundane code has been removed.